### PR TITLE
Move Webui::Users::AnnouncementsController to Bootstrap

### PR DESCRIPTION
**Pull request by vpereira**
Move the create action to bootstrap.

### CODE REVIEW

**Comment by vpereira** [comment]
[Webui::UserController](https://trello.com/c/zOHXNyMW/1261-webuiusercontroller)

File: src/api/app/controllers/webui/users/announcements_controller.rb [hunk 1]
```diff
@@ -9,6 +9,5 @@ def create
     else
       flash.now[:error] = "Couldn't find Announcement"
     end
-    switch_to_webui2
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/users/announcements_controller.rb]: This should be kept until we get rid of it completely (which we will do later).

**Comment by vpereira** [comment]
if you want to create a Announcement via osc:

```
osc -A  https://obs-reviewlab.opensuse.org/vpereira-move_announcements_controllers_to_bootstrap/announcements  -X POST -d '<announcement><title>foo</title><content>bar</content></announcement' /announcements
```

File: src/api/app/controllers/webui/users/announcements_controller.rb (hunk 1 above)
Comment (vpereira) [reply src/api/app/controllers/webui/users/announcements_controller.rb]: why if it's not necessary? It's just necessary if it needs the webui2 views in the view path...

**Comment by dmarcoux** [comment]
I fixed the command in @vpereira's comment.

**Pull request merged**